bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -45,197 +41,74 @@
-</p>
-
-<h3>Easy to control</h3>
-
-<p>
-Tiles are easily pinned, moved around or removed using simple drag-and-drop and close interfaces.
-If you do not want to see any Tiles, you can deactivate them completely in two clicks through the new tab gear control.
-</p>
-
-{% endif %}
-
-<h3>{{ _('Respects your privacy') }}</h3>
-
-<p>
-{% if LANG in ['en-US'] %}
-Tiles are a part of our initiative to set new standards for Internet advertisers and content publishers in the collection and use of personal data.
-</p>
-<p>
-Mozilla respects your privacy and any data collection by Mozilla via Tiles will be done in accordance with our <a href="{{ url('privacy.notices.firefox') }}">Privacy Notice</a>.
-The data we collect is to help us deliver the most interesting and relevant content to our users.
-For more on what data Mozilla collects via Tiles, see below.
-{% else %}
-{{ _('As with all Mozilla products, Tiles provide users with extensive control over their personal data.') }}
-{{ _('Data collection is limited to the aggregate number of impressions, clicks and Tile placement.') }}
-{{ _('Nothing is ever personally identifiable. In other words, we don’t know who clicks on what.') }}
-{{ _('We just know the total number of views and clicks an individual Tile receives. This helps us deliver the most interesting and relevant content.') }}
-{{ _('You can deactivate click and impression tracking by clicking on the gear icon on the top right corner of the new tab page, and selecting Classic to show only unenhanced History Tiles or Blank mode.') }}
-{% endif %}
-</p>
-
-{% endblock %}
-
-{# The contents of the dashboard_faq block appears below the intro. For each question/answer
-   use the following format:
-
-    <section>
-      <h3 data-accordion-role="tab">{{ _('Question title goes here?') }}</h3>
-      <div data-accordion-role="tabpanel">
-        <p>
-          {% trans %}
-            Question answer goes here.
-          {% endtrans %}
-        </p>
-      </div>
-    </section>
-
-#}
-
-{% block dashboard_faq %}
-<section id="faq" class="accordion accordion-auto-init zebra">
-
-<h2>{{ _('Frequently Asked Questions') }}</h2>
-
-<section>
-  <h3 data-accordion-role="tab">{{ _('Are you tracking users now?') }}</h3>
-  <div data-accordion-role="tabpanel">
-    <p>
-      {% trans %}
-        We only collect data from the New Tab page in aggregate form, meaning no data is personally identifiable.
-      {% endtrans %}
-    </p>
-  </div>
+<section class="tls-personalize">
+  <h2>{{ _('Personalize your New Tab experience') }}</h2>
+  <p>
+    {{ _('You are in complete control of your New Tab page.') }}
+    {{_('Customize your experience by rearranging, pinning or removing sites, and set up New Tab just the way you want it.') }}
+  </p>
+  <ol>
+    <li>{{ _('Drag a site to exactly where you want it.') }}
+      <img src="{{ static('img/firefox/tiles/drag-site.png') }}" alt="{{ _('Screenshot') }}">
```

Comment:
If this alt text can't be descriptive of what the image shows, I think an empty alt would be better than just "screenshot." I think the captions alone are good enough.